TellingMe/tellingMe/presentation/signUp/viewControllers/SignUpViewController.swift

```diff
@@ -0,0 +1,268 @@
+//
+//  SignUpViewController.swift
+//  tellingMe
+//
+//  Created by 마경미 on 27.09.23.
+//
+
+import UIKit
+
+import RxCocoa
+import RxSwift
+import SnapKit
+import Then
+
+final class SignUpViewController: BaseViewController {
+    private let viewModel = SignUpViewModel()
+
+    private let disposeBag = DisposeBag()
+    
+    private let headerView = LogoHeaderView()
+    private let skipButton = UIButton()
+    private let progressView = GradientProgressBar(progressViewStyle: .bar)
+    private let containerView = UIView()
+    private let pageViewController = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal)
+    private let leftButton = SecondaryIconButton()
+    private let rightButton = SecondaryIconButton()
+    private let infoview = BottomInfoView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        bindViewModel()
+        setLayout()
+        setStyles()
+        setPageViewController()
+    }
+    
+    deinit {
+        print("SignUpViewController Deinited")
+    }
+}
+
+extension SignUpViewController {
+    private func bindViewModel() {
+        skipButton.rx.tap
+            .throttle(.milliseconds(1000), scheduler: MainScheduler.instance)
```

Comment:
p1
`throttle()` 에 parameter 중에서, latest 라는 parameter 가 있습니다.
`.throttle(.seconds(3), latest: false)` 이런 형식인데요.
`latest` 의 default 값은 `true` 입니다. 즉, 1000밀리초(1초) 내로 두 번을 누르게 되면 latest tap 값이 한번 더 들어가서 총 2번이 찍히게 됩니다. 1초 내로 3번을 눌러도 2번이 들어갑니다. 하지만, 저희는 중복 값을 막는것이 목적이므로 아마 빠르게 두번 누르시면 페이지 이동이 한번 더 되는걸 볼 수 있으실겁니다! 설령 그렇지 않더라도 ux 상의 버그가 언제든 생길 수 있는 컴포넌트이기 때문에 false 로 바꾸는걸 추천 드립니다!
```suggestion
            .throttle(.milliseconds(1000), latest: false, scheduler: MainScheduler.instance)
```